src/components/apps/launch/ReferenceGenomeSelect.js

```diff
@@ -53,6 +57,7 @@ const ReferenceGenomeSelect = ({ param, ...props }) => {
             margin="normal"
             {...selectProps}
         >
+            {!selectProps.required && <MenuItem value="">&nbsp;</MenuItem>}
```

Comment:
This 1-liner allows users to clear a selection when the param is not required, but also allows app integrators to clear a default value setting.

It's the same strategy used to allow app integrators to clear the default selection for `List` params.

Though I just realized that users cannot clear a selection for optional `List` params in the App Launch form. I can also add that improvement to this PR.

Reply:
✅ done

Reply:
✅ done